<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QMainWindow" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>电气火灾监控远程客户端</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget{
	background-color: rgb(15, 41, 66);
	border-image: url(:/Image/widget_1.png);
}</string>
   </property>
   <widget class="QWidget" name="widgetFunBar" native="true">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>60</y>
      <width>761</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widgetFunBar{
background-color: rgba(10, 89, 111, 50);
border:1px solid;
border-radius:30px;
}


QToolButton{
font: 16pt &quot;微软雅黑&quot;;
border:2px none;
color: rgb(255, 255, 255);
border-color: rgb(0, 170, 127);
}

QToolButton:hover{
font: 16pt &quot;微软雅黑&quot;;
color: rgb(255, 0, 0);
color: rgb(0, 185, 185);
/*background-color: rgb(3, 127, 119);*/

}</string>
    </property>
    <widget class="QToolButton" name="tBtnMornitor">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>5</y>
       <width>140</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>监控信息</string>
     </property>
     <property name="icon">
      <iconset resource="../image.qrc">
       <normaloff>:/Image/connect.png</normaloff>:/Image/connect.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="tBtnSystemSet">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>5</y>
       <width>140</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>网络设置</string>
     </property>
     <property name="icon">
      <iconset resource="../image.qrc">
       <normaloff>:/Image/selfCheck.png</normaloff>:/Image/selfCheck.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="tBtnRecord">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>5</y>
       <width>140</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>记录查询</string>
     </property>
     <property name="icon">
      <iconset resource="../image.qrc">
       <normaloff>:/Image/dataSearch.png</normaloff>:/Image/dataSearch.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="tBtnSystemInfo">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>380</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>系统信息</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/Image/8.ico</normaloff>:/Image/8.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="tBtnClose">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>5</y>
       <width>140</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭软件</string>
     </property>
     <property name="icon">
      <iconset resource="../image.qrc">
       <normaloff>:/Image/power.png</normaloff>:/Image/power.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="tBtnUser">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>140</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>用户登录</string>
     </property>
     <property name="icon">
      <iconset resource="../image.qrc">
       <normaloff>:/Image/user.png</normaloff>:/Image/user.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>1555</width>
      <height>770</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
font: 16pt &quot;微软雅黑&quot;;
border:1px solid;
border-color: rgb(0, 0, 0);
color: rgb(44, 44, 44);
border-radius:20px;
}
QGroupBox::title {
color: rgb(255, 255, 255);
subcontrol-origin: margin;
subcontrol-position: top left; /* position at the top center */
padding: 0 3px;
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(28, 183, 200, 255), stop:1 rgba(0, 85, 127, 255));
  }</string>
    </property>
    <property name="title">
     <string>监控信息                                                                                                                                                  </string>
    </property>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>35</y>
       <width>1540</width>
       <height>730</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <widget class="QLabel" name="lbSystemTime">
     <property name="geometry">
      <rect>
       <x>1295</x>
       <y>15</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>2018年09月10日/16:20:30</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>1255</x>
       <y>15</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Image/clock.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>0</y>
       <width>440</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 30pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>电气火灾远程客户端</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>15</y>
       <width>311</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;微软雅黑&quot;;
color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>西安盛赛尔电子有限公司</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>10</y>
       <width>151</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;楷体&quot;;
border-image: url(:/Image/ss_logo.png);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>widget</zorder>
   <zorder>widgetFunBar</zorder>
   <zorder>groupBox</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
